src/public/js/util.js

```diff
@@ -38,6 +38,12 @@ var UTIL = (function() {
      0xb4, 0x7c, 0xb8, 0x26, 0x77, 0x99, 0xe3, 0xa5,
      0x67, 0x4a, 0xed, 0xde, 0xc5, 0x31, 0xfe, 0x18,
      0x0d, 0x63, 0x8c, 0x80, 0xc0, 0xf7, 0x70, 0x07]);
+  log3.get = function(x) {
```

Comment:
you are correct! 

You can check if an object has a field in two ways -- if you want to check whether the field is a direct property of the object, you can use [Object.hasOwnProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty), and if you want to test if the field is in the object's prototype chain (the prototype chain is the Javascript mechanism for inheritance, basically) you can use the `in` keyword.

So for example -- let's say we made an empty array then added a `get` method to it --
```
var test = [];
test.get = function() {...};
```

here's what you'd find:
```
// these would all evaluate to true
test.hasOwnProperty('get');
'get' in test;
'hasOwnProperty' in test;

// this would be false
test.hasOwnProperty('hasOwnProperty'); 
```

The `hasOwnProperty` example is the interesting one -- `'hasOwnProperty' in test;` being true tells you that `test.hasOwnProperty` exists; `test.hasOwnProperty('hasOwnProperty');` being false tells you that `hasOwnProperty` is defined somewhere up the prototype chain from `test` -- it's not directly a property of `test`, it's inherited.